src/mmw/js/src/modeling/models.js

```diff
@@ -751,6 +758,7 @@ var ScenariosCollection = Backbone.Collection.extend({
             aoi_census: aoi_census
         });
 
+        scenario.waitToRefresh = true;
```

Comment:
This is not actually meant to be an official attribute, it is just an extra piece of state.  If the consensus is that it should be an attribute, then I can make it so.